### PR TITLE
Add org.gtk.Gtk3theme.Orchis

### PR DESCRIPTION
**Pull request by rguards**
### Please confirm your submission meets all the criteria

Orchis is a [Material Design](https://material.io/) theme for GNOME/GTK based desktop environments.
- [x] Please describe your application briefly.
- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/vinceliuice/Orchis-theme/issues/432
- [ ] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

**Comment by rguards** [comment]
bot, build io.github.cleomenezesjr.aurea

**Comment by rguards** [comment]
bot, build org.gtk.Gtk3theme.Orchis

**Comment by rguards** [comment]
bot, build org.gtk.Gtk3theme.Orchis

**Comment by hfiguiere** [comment]
This is missing

> Please describe your application briefly.

File: org.gtk.Gtk3theme.Orchis.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+          url: https://api.github.com/repos/vinceliuice/Orchis-theme/releases/latest
+          tag-query: .tag_name
+          version-query: "$tag"
+          url-query: .tarball_url
+
+  - name: appdata
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 ${FLATPAK_ID}.appdata.xml -t ${FLATPAK_DEST}/share/appdata
+      - appstream-compose --basename=${FLATPAK_ID} --prefix=${FLATPAK_DEST} --origin=flatpak ${FLATPAK_ID}
```
Comment (hfiguiere) [inline org.gtk.Gtk3theme.Orchis.yml:35]: this is no longer necessary. Also delete line 8.

**Comment by rguards** [comment]
> This is missing
> 
> > Please describe your application briefly.

OK, I completed it.  I was careless.

File: org.gtk.Gtk3theme.Orchis.yml (hunk 1 above)
Comment (rguards) [reply org.gtk.Gtk3theme.Orchis.yml]: I removed it.

File: org.gtk.Gtk3theme.Orchis.appdata.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="runtime">
+  <id>org.gtk.Gtk3theme.Orchis</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <name>Orchis Gtk Theme</name>
+  <summary>Material Design theme for GNOME/GTK based desktop environments</summary>
+  <description>
+      <p>Material Design theme for GNOME/GTK based desktop environments</p>
+  </description>
+  <url type="homepage">https://github.com/vinceliuice/Orchis-theme</url>
```
Comment (hfiguiere) [inline org.gtk.Gtk3theme.Orchis.appdata.xml:10]: you should ad release information to at least display a version.

Comment (rguards) [reply org.gtk.Gtk3theme.Orchis.appdata.xml:10]: It was added now.

**Comment by rguards** [comment]
bot, build org.gtk.Gtk3theme.Orchis

**Comment by rguards** [comment]
This GTK theme is works well now. GTK apps can use this theme on my GNOME desktop.

**Comment by bbhtt** [comment]
>  I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. Link:

Can you please open an issue [upstream](https://github.com/vinceliuice/Orchis-theme/issues) that you are packaging this for Flathub? Thanks

**Comment by rguards** [comment]
> > I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. Link:
> 
> Can you please open an issue [upstream](https://github.com/vinceliuice/Orchis-theme/issues) that you are packaging this for Flathub? Thanks

OK, I opened a new issue

**Comment by rguards** [comment]
The author agreed this submission https://github.com/vinceliuice/Orchis-theme/issues/432#issuecomment-2097167248

**Comment by bbhtt** [comment]
Yes sorry the previous comment went out of my radar.

**Pull request closed without merging**